pyth-sdk-solana/Cargo.toml

```diff
@@ -24,6 +24,10 @@ pyth-sdk = { path = "../pyth-sdk", version = "0.7.0" }
 [dev-dependencies]
 solana-client = ">= 1.9, < 1.15"
 solana-sdk = ">= 1.9, < 1.15"
+pyth-oracle = { path = "../../pyth-client/program/rust", features = ["library"] }
```

Comment:
Relative path just for now because it relies on the upstream PR being merged first.